troubadix/standalone_plugins/changed_cves.py

```diff
@@ -0,0 +1,98 @@
+import re
+from argparse import ArgumentParser, Namespace
+from subprocess import CalledProcessError
+
+from pontos.terminal.terminal import ConsoleTerminal
+
+from troubadix.argparser import file_type
+from troubadix.helper.patterns import (
+    SpecialScriptTag,
+    get_special_script_tag_pattern,
+)
+from troubadix.standalone_plugins.common import get_merge_base, git
+
+CVE_PATTERN = re.compile(r"CVE-\d{4}-\d{4,7}")
+
+
+def compare(old_content: str, content: str):
+    old_cves = get_cves_from_content(old_content)
+    cves = get_cves_from_content(content)
+
+    missing_cves = sorted(old_cves.difference(cves))
+    new_cves = sorted(cves.difference(old_cves))
+
+    return missing_cves, new_cves
```

Comment:
```suggestion
    old_cves = get_cves_from_content(old_content)
    current_cves = get_cves_from_content(content)

    missing_cves = sorted(old_cves.difference(current_cves))
    added_cves = sorted(current_cves.difference(old_cves))

    return missing_cves, added_cves
```
When 4 variables are named *_cves, I feel this is a bit more clear

Reply:
Fixes + some additional renamings